### PR TITLE
Update GitHub categories and add ibm security logout

### PR DESCRIPTION
**Pull request by c00crane**
Update GitHub test categories:  Move saml projects out of Quarantine (and Security_5) and create new test categories: SECURITY_SAML_1, SECURITY_SAML_2, SECURITY_SAML_3, SECURITY_SAML_4
Add the copyrights to the gradle.properties files in the saml FAT projects
Fix the typo in the gradle property httpServletReuqestAppWar - (should be httpServletRequestAppWar) - this isn't breaking anything as the property definitions and the users are all using the same bad spelling... 
Port the project: com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout - I missed this when porting all of the others - it is built on top of com.ibm.ws.security.saml.sso_fat.logout

### CODE REVIEW

**Comment by c00crane** [comment]
#build

File: dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/.project [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout</name>
+	<comment></comment>
+	<projects>
+		<project>com.ibm.ws.security_fat.common</project>
+		<project>com.ibm.ws.security.saml.sso_fat.common</project>
+		<project>com.ibm.ws.security.saml.sso_fat.logout</project>
```
Comment (ayoho) [inline dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/.project:8]: Never noticed _that_ in .project files before.

File: dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+fat.project: true
+
+tested.features: jsp-2.3, jdbc-4.1, el-3.0, servlet-4.0, samlweb-2.0, jdbc-4.0
```
Comment (ayoho) [inline dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/bnd.bnd:19]: There are tests using jdbc-4.x in this bucket?

File: dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/build.gradle [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    maven {
+      credentials {
+        username userProps.getProperty("artifactory.download.user")
+        password userProps.getProperty("artifactory.download.token")
+      }
+      url ("https://" + gradle.userProps.getProperty("artifactory.download.server") + "/artifactory/wasliberty-maven-remote")
+    }
+  } else {
+    mavenCentral()
+  }
```
Comment (ayoho) [inline dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/build.gradle:24]: Is this all needed?

**Review by ayoho — APPROVED** [review]

**Comment by c00crane** [comment]
run-libby-bot

File: dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/bnd.bnd (hunk 2 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/bnd.bnd:19]: I had to add those to satisfy the overall fat tooling (it spits out failures when it thinks features are not listed that are being used).

File: dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/build.gradle (hunk 3 above)
Comment (c00crane) [reply dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/build.gradle:24]: I copied what was done for com.ibm.ws.security.sso_fat.
I think I omitted it when I ported com.ibm.ws.security.sso_fat.2 and had problems - I can try to remove it (from this and possibly all of the other saml fats...)

**Comment by c00crane** [comment]
#run-libby-bot

**Pull request merged**